src/main/java/org/apache/onami/persist/TransactionalAnnotationReader.java

```diff
@@ -60,7 +60,7 @@ Transactional readAnnotationFrom(MethodInvocation methodInvocation) {
   /**
    * Helper class for obtaining the default of {@link Transactional @Transactional}.
    */
-  @Transactional
+  @Transactional(onUnits = {})
```

Comment:
This seems super weird. Where is `DefaultTransactional` used? It might just not be necessary at all anymore.